File: scratch.js
const https = require('https');
const options = {
  hostname: 'api.airtable.com',
  path: '/v0/appkRWtF2j99XgBTq/tblUYmhOvtHGJ9NO3?maxRecords=1',
  headers: {
    'Authorization': 'Bearer patM0xqBHeF8WJIts'
  }
};
https.get(options, (res) => {
  let data = '';
  res.on('data', chunk => data += chunk);
  res.on('end', () => {
    console.log(data);
  });
});

File: scratch.py
import urllib.request, json
req = urllib.request.Request('https://api.airtable.com/v0/appkRWtF2j99XgBTq/tblUYmhOvtHGJ9NO3?maxRecords=1&returnFieldsByFieldId=true', headers={'Authorization': 'Bearer patM0xqBHeF8WJIts'})
res = urllib.request.urlopen(req)
data = json.loads(res.read())
print(json.dumps(data, indent=2))
